requirements.txt

```diff
@@ -1,30 +1,37 @@
-certifi==2024.8.30
+#
```

Comment:
can you omit this header? 
I believe that if you run `tox -e upgrade` this is not included(?)